core/src/com/po/fuck/collections/EntityCollection.java

```diff
@@ -0,0 +1,20 @@
+package com.po.fuck.collections;
+
+import com.po.fuck.Entity;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public final class EntityCollection extends SimpleCollection<Entity> {
+    EntityCollection() {
+        super(Entity.class);
+    }
+
+    public List<Entity> getTeam (int teamTag) {
+        return objects.stream().filter(o -> o.getTeamTag() == teamTag).collect(Collectors.toList());
```

Comment:
Replace `==` with `.equals()`. As in the future the tags could be replaced with another Class.

Reply:
If something will be replaced in the future, then there will be some refactor, so the person who replaces it, will fix it also.
For now I don't see any reason for it.